InstallNotes/MakeDefsLocalExamples/Mac-Catalina-GCC.Make.defs.local

```diff
@@ -0,0 +1,34 @@
+# This make defs file is for Mac OS Catalina
+# It assumes that you have used homebrew to install
+# the relevant packages as indicated for the commands below
+# You should also have obtained up to date Xcode command line tools
+# using 'xcode-select --install'
+DIM             = 3
+DEBUG           = TRUE
+OPT             = TRUE
+PRECISION       = DOUBLE
+USE_64          = TRUE
+# brew install gcc
+CXX             = g++-9 -std=c++14
+FC              = gfortran-9
+# brew install libomp
+OPENMPCC        = TRUE
+# brew install mpich
+MPI             = TRUE
+MPI_DIR         = /usr/local/Cellar/mpich/3.3.2
+MPICXX          = g++-9 -std=c++14 -I${MPI_DIR}/include -lmpi -fopenmp
+# brew install hdf5 is fine for serial builds
+USE_HDF         = TRUE
+HDF5_DIR = /usr/local/Cellar/hdf5/1.12.0
+HDFINCFLAGS     = -I${HDF5_DIR}/include
+HDFLIBFLAGS     = -L${HDF5_DIR}/lib -lhdf5 -lz
+# hdf5 v1.10 library downloaded and built using CMake with --enable-parallel
+# (see also the hdf5_cmake_configure.sh script)
+HDF5_PARALLEL_DIR = /usr/local/Cellar/hdf5-parallel
+HDFMPIINCFLAGS  = -I${HDF5_PARALLEL_DIR}/include
+HDFMPILIBFLAGS  = -L${HDF5_PARALLEL_DIR}/lib -lhdf5 -lz
+cxxoptflags     = -march=native -O3 -ftemplate-depth-150
+foptflags       = -march=native -O3
+lddbgflags      = -framework Accelerate -lgfortran 
+ldoptflags      = -framework Accelerate -lgfortran
+syslibflags     = -lblas -llapack -L${MPI_DIR}/lib
```

Comment:
I think there might be a bit of redundancy here since you're passing the same flags to the linker in the DEBUG and OPT case. Can you try to see if the following works instead?
```suggestion
syslibflags     = -framework Accelerate -lgfortran -lblas -llapack -L${MPI_DIR}/lib
```
P.S. I had no idea about macOS "frameworks" before I saw this and looked it up. This isn't a concept on Linux...

Reply:
Also, is the `-lgfortran` flag definitely needed?